### PR TITLE
Feat/issue 72 default crosshair

### LINKED ISSUE #72: [Feature] provide `crosshair` by default for some charts
### What problem does this feature solve?

Like lineChart, barChart, we should provide crosshair component by default.

### What does the proposed API look like?

none

### PR DESCRIPTION
**Pull request by kkxxkk2019**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [x] Enhancement feature
- [ ] Refactoring]
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close #72 

### 💡 Background and solution

Add a default crosshair configuration for some Cartesian charts.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

**Review by xuefei1313 — APPROVED** [review]

**Comment by xuefei1313** [comment]
Why not add it uniformly under the cartesian chart?

**Comment by kkxxkk2019** [comment]
> Why not add it uniformly under the cartesian chart?
because some charts like heatmap、linearprogress and scatter is Inappropriate

**Comment by xiaoluoHe** [comment]
Some polar charts like radar chart can also set default crosshair?
Maybe u can create another issue~

**Comment by kkxxkk2019** [comment]
> Some polar charts like radar chart can also set default crosshair? Maybe u can create another issue~

OK，I will add in another PR ~

File: packages/vchart/src/chart/util.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        label: {
+          visible: false
+        },
+        line: {
+          visible: true,
+          type: 'rect'
+        }
+      }
+    },
+    spec.crosshair
```
Comment (xiaoluoHe) [inline packages/vchart/src/chart/util.ts:19]: here, I think `spec.crosshair` might be an array.
image

Comment (kkxxkk2019) [reply packages/vchart/src/chart/util.ts]: I have handled this, see https://github.com/VisActor/VChart/pull/83/commits/6ec9369b5c47fdb729f7d077bb0b3ae5bcda5a86

**Review by xiaoluoHe — APPROVED** [review]

**Review by xiaoluoHe — APPROVED** [review]

**Pull request merged**